static/game/pickup.js

```diff
@@ -80,7 +93,7 @@ export class Pickup extends Entity {
       /** @type {Hero} */ (entity).addBombs(1);
     } else if (this.pickupType === PickupEnum.health) {
       playSound("health-pickup");
-      /** @type {Hero} */ (entity).gainHealth(5);
+      /** @type {Hero} */ (entity).gainHealth(10);
```

Comment:
nah dude! too easy

Reply:
10 is the basic unit of enemy damage so it feels cheap to make the health pickups heal only half that.

Reply:
ok